frontend/components/Domain/Recipe/RecipeScaleEditButton.vue

```diff
@@ -1,16 +1,12 @@
 <template>
-  <div>
+  <div v-if="yieldDisplay">
     <div class="text-center d-flex align-center">
       <div>
-        <v-menu v-model="menu" :disabled="!editScale" offset-y top nudge-top="6" :close-on-content-click="false">
+        <v-menu v-model="menu" :disabled="!canEditScale" offset-y top nudge-top="6" :close-on-content-click="false">
           <template #activator="{ on, attrs }">
             <v-card class="pa-1 px-2" dark color="secondary darken-1" small v-bind="attrs" v-on="on">
-              <span v-if="!recipeYield"> x {{ scale }} </span>
-              <div v-else-if="!numberParsed && recipeYield">
-                <span v-if="numerator === 1"> {{ recipeYield }} </span>
-                <span v-else> {{ numerator }}x {{ scaledYield }} </span>
-              </div>
-              <span v-else> {{ scaledYield }} </span>
+              <!-- eslint-disable-next-line vue/no-v-html -->
```

Comment:
Can we add an edit icon to this? 
Had user feedback that it is not intuitive that this can be clicked to open the scaling menu. 

This would help people figure this out easier.